### PR TITLE
fixes vitamin making you fat

### PR DESCRIPTION
**Pull request by RafRoq**
:cl: vitamin does not make you fat anymore as intended
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -1,2 +1,7 @@
 /datum/reagent/consumable/condensedcapsaicin/reaction_turf(turf/T, reac_volume)
   return
+
+/datum/reagent/consumable/nutriment/vitamin/on_mob_life(mob/living/carbon/M)
+	if(M.satiety < 569)
+		M.satiety += 30
+	. = ..()
```
Comment (tortellinitony) [inline hippiestation/code/modules/reagents/chemistry/reagents/food_reagents.dm]: `return ..()`

**Review by tortellinitony — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/reagents/chemistry/reagents/food_reagents.dm (hunk 1 above)
Comment (RafRoq) [reply hippiestation/code/modules/reagents/chemistry/reagents/food_reagents.dm]: What is the function of return here

**Comment by Carbonhell** [comment]
Code's wrong. This is already implemented here, aswell:
https://github.com/HippieStation/HippieStation/blob/master/code/modules/reagents/chemistry/reagents/food_reagents.dm#L96
Is 600 too much?Are you actually sure it makes you fat?

**Comment by RafRoq** [comment]
@Carbonhell  above 600 you are fat and slowed, if you have 599 nutriment it will pump more 30 and keep you fat changing it to 570 will make that vitamin will never make you fat and slow

**Comment by Carbonhell** [comment]
Yea but you are calling the tg definition with ..(), so this code will just cause people to be filled twicely. You need to edit the value in the tg file directly.

**Comment by Carbonhell** [comment]
Stale.

**Pull request closed without merging**